contracts/MetadataViews.cdc

```diff
@@ -695,6 +695,28 @@ pub contract MetadataViews {
         }
     }
 
+    // Taken from GaiaPackNFT https://flow-view-source.com/mainnet/account/0xfdae91e14e960079/contract/GaiaPackNFT
+    // AllDay Packs have a subset of these so it should be safe to use this more verbose set
+    pub enum PackStatus: UInt8 {
+        pub case Sealed
+        pub case RevealRequested
+        pub case Revealed
+        pub case OpenRequested
+        pub case Opened
+    }
```

Comment:
I'm not sure if an enum is the right call given its finality. We could instead make it a string so we can add to this as needed. Both Gaia and AD packs use an enum, though

Reply:
Could we potentially make it an integer, and then have a switch statement to indicate what its status is? that way we can add more to it if needed, but it is still similar to an enum

Reply:
No strong feelings on that part either way. I was talking to Chris on the Gaia team and he had suggested maybe going with a "Consumable" standard instead where packs are just a one-time consumable. In which case, we would amend this to let you ask how many charges are left in some capacity.

I'll open a ticket to discuss this and then we can amend this PR as needed, it is just a starting point for discussion 🙂

Reply:
https://github.com/onflow/flow-nft/issues/149